### PR TITLE
MNT: Add noarch: python

### LINKED ISSUE #1: @conda-forge-admin, please add noarch: python
we do not need a build for every python version.

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've made the recipe `noarch: python` as instructed in #1.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

Fixes #1

### CODE REVIEW

**Comment by looooo** [comment]
@conda-forge/core Why is python a dependency if it is listed in the test-requirements only? Removing the python dependency from the test-requirements solves this issue.

**Comment by beckermr** [comment]
This recipe cannot be noarch as it is using compilers.

**Comment by looooo** [comment]
@beckermr correct, but there are 3 builds created because python is in the test-requirements. But having python in the test requirements doesn't mean that the package is depending on python. I will remove the test for now.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,12 +38,12 @@ test:
   requires:
     - python
```
Comment (xhochy) [inline recipe/meta.yaml]: Might help here
```suggestion
    - python *
```

Comment (looooo) [reply recipe/meta.yaml]: great thanks @xhochy

**Pull request merged**